### PR TITLE
Fix for removal of participant_name from ag_kit_barcodes

### PR DESCRIPTION
**Pull request by squirrelo**
This updates labadmin to comply with the database change in american gut centralizing the participant_name column.

### CODE REVIEW

File: knimin/lib/tests/test_data_access.py [hunk 1]
```diff
@@ -121,6 +121,99 @@ def test_get_access_levels(self):
...
+        obs = db.participant_names()
+        self.assertEqual(len(obs), 8237)
+        self.assertEqual(obs[0], ['000027561', 'REMOVED-0'])
+
+    def test_search_barcodes(self):
+        obs = db.search_barcodes('000001124')
+        self.assertEqual(obs, ['d8592c74-7c27-2135-e040-8a80115d6401'])
+
+        obs = db.search_barcodes('REMOVED-8')
+        exp = ['d8592c74-9491-2135-e040-8a80115d6401',
```
Comment (josenavas) [inline knimin/lib/tests/test_data_access.py]: There are a few values repeated in here. Does that mean that there is a distinct missing in a query?

Comment (squirrelo) [inline knimin/lib/tests/test_data_access.py]: yup, distinct added.

**Comment by squirrelo** [comment]
Comment addressed

**Comment by josenavas** [comment]
👍

**Pull request merged**